program/c/src/oracle/model/test_price_model.c

```diff
@@ -14,10 +14,7 @@ qcmp( void const * _p,
   return 0;
 }
 
-int
-main( int     argc,
-      char ** argv ) {
-  (void)argc; (void)argv;
+int test_price_model() {
```

Comment:
the changes in the C test files are:
1. export a function name so we can call them from rust (instead of main)
2. delete the noisy printf statements
3. I reduced the iteration counts on a couple of the randomized tests, because they took several minutes to run.